experimental/QuadFormAndIsom/src/QuadFormAndIsom.jl

```diff
@@ -0,0 +1,13 @@
+function set_lwi_level(k::Int)
```

Comment:
Is this `lwi` here an abbreviation of something? Maybe change it to a more meaningful name

Reply:
It is not meant to stay so forever, but only during the "experimental part" of this code. `lwi` means "lattice with isometry", which was the original name of the project. I would rather keep it like that because it is short enough, and the meaning is not really important compared to what it is used for (the documentation currently covers the use of the function.